src/test/java/teammates/test/cases/browsertests/InstructorSearchPageUiTest.java

```diff
@@ -109,7 +111,32 @@ private void testSearch() throws Exception {
         searchPage.verifyHtmlMainContent("/instructorSearchPageSearchStudentsForStudent2.html");
     }
 
+    private void testDeleteAction() throws Exception {
+
+        ______TS("action: delete");
+
+        searchPage.clearSearchBox();
+        String searchContent = "\"student2 2 In Course1\"";
+        searchPage.inputSearchContent(searchContent);
+        searchPage.clickSearchButton();
+
+        String studentName = testData.students.get("student2.2InCourse1").name;
+        String studentEmail = testData.students.get("student2.2InCourse1").email;
+        String courseId = testData.courses.get("typicalCourse1").getId();
+
+        searchPage.clickDeleteAndCancel(courseId, studentName);
+        assertNotNull(BackDoor.getStudent(courseId, studentEmail));
+
+        String expectedStatus = Const.StatusMessages.STUDENT_DELETED;
+        searchPage.clickDeleteAndConfirm(courseId, studentName);
+        InstructorCourseDetailsPage courseDetailsPage = searchPage.changePageType(InstructorCourseDetailsPage.class);
+        courseDetailsPage.waitForTextsForAllStatusMessagesToUserEquals(expectedStatus);
```

Comment:
Can just use `Const.StatusMessages.STUDENT_DELETED` and get rid of `expectedStatus`.

Reply:
Ohh..my bad, I don't know why I missed that out.